cli/provision.go

```diff
@@ -20,13 +22,41 @@ import (
 const jsonExt = ".json"
 const csvExt = ".csv"
 
+const csvThingsFieldCount = 4
+
+// These constants define the order of the CSV columns (fields) of records containing Things to be provisioned
+const (
+	csvThingsFieldID = iota
+	csvThingsFieldName
+	csvThingsFieldGroupID
+	csvThingsFieldProfileID
+)
+
+const csvProfilesFieldCount = 12
+
+// These constants define the order of the CSV columns (fields) of records containing Profiles to be provisioned
+const (
+	csvProfilesFieldID = iota
+	csvProfilesFieldName
+	csvProfilesFieldGroupID
+	csvProfilesFieldConfigContentType
+	csvProfilesFieldConfigWrite
+	csvProfilesFieldConfigWebhook
+	csvProfilesFieldConfigTransformerDataFilters
+	csvProfilesFieldConfigTransformerDataField
+	csvProfilesFieldConfigTransformerTimeField
+	csvProfilesFieldConfigTransformerTimeFormat
+	csvProfilesFieldConfigTransformerTimeLocation
+	csvProfilesFieldConfigSMTPID
```

Comment:
This names are too long. You can probably shorter them by using singular, removing `csv` prefix and `Field`. 
For example, `profileConfTransformerDataFilters`.